tutor/jest-playwright.config.js

```diff
@@ -0,0 +1,22 @@
+const serverPort = process.env.SERVER_PORT || 8110;
+
+const DEBUG = Boolean(process.env.DEBUG || false);
```

Comment:
this lets you run `DEBUG=1 yarn run test tutor:test:e2e` and visualize the browser progress.

You can put a await jestPlaywright.debug() in the test to pause execute https://github.com/playwright-community/jest-playwright#put-in-debug-mode